### PR TITLE
More pyppeteer tests

### LINKED ISSUE #438: Add another end-to-end test to mark a scan and export it as CSV

### PR DESCRIPTION
**Pull request by annehaley**
Resolves #438

### CODE REVIEW

File: miqa/core/tests/pyppeteer/test_scan_upload.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    await file_input.uploadFile(
+        'samples/Demo Project/IXI002/0828-DTI/IXI002-Guys-0828-DTI-00.nii.gz',
+        'samples/Demo Project/IXI002/0828-DTI/IXI002-Guys-0828-DTI-01.nii.gz',
+        'samples/Demo Project/IXI002/0828-DTI/IXI002-Guys-0828-DTI-02.nii.gz',
+    )
+    await page.waitFor(3_000)
+    await (
+        await page.waitForXPath('//span[contains(@class, "v-btn__content")][contains(.,"Upload")]')
+    ).click()
+    await page.waitFor(10_000)
```
Comment (zachmullen) [inline miqa/core/tests/pyppeteer/test_scan_upload.py]: Does this actually need to be 10 seconds, or was your issue earlier a red herring?

**Review by zachmullen — APPROVED** [review]

File: miqa/core/tests/pyppeteer/test_scan_upload.py (hunk 1 above)
Comment (annehaley) [reply miqa/core/tests/pyppeteer/test_scan_upload.py]: Looks like the latter! I cut the wait to 5 seconds instead.

File: miqa/core/tests/pyppeteer/test_save_decisions.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    await (
+        await page.waitForXPath(
+            '//span[contains(@class, "v-btn__content")][contains(.,"Questionable")]'
+        )
+    ).click()
+    await page.waitFor(1_000)
+
+    # Mark all artifacts as present on the third scan and mark it as questionable
+    artifact_chips = await page.xpath('//span[contains(@class,"v-chip__content")]')
+    [(await chip.click()) for chip in artifact_chips]
```
Comment (zachmullen) [inline miqa/core/tests/pyppeteer/test_save_decisions.py]: This is a minor nit, but this pattern is discouraged in python.
```suggestion
    for chip in artifact_chips:
        await chip.click()
```

Basically, you should only use list comprehension syntax if you are interested in its return value, not for side effects.

File: miqa/core/tests/pyppeteer/test_save_decisions.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+        '//div[contains(., "No comment")]/following-sibling::div/'
+        'span/span[contains(@class,"v-chip__content")]'
+    )
+    assert len(saved_artifacts) == len(artifact_chips)
+
+    # Submit another decision on the third scan
+    # This time marking all artifacts as absent and the scan as Usable
+    artifact_chips = await page.xpath('//span[contains(@class,"v-chip__content")]')
+    [(await chip.click()) for chip in artifact_chips]
+    [(await chip.click()) for chip in artifact_chips]
```
Comment (zachmullen) [inline miqa/core/tests/pyppeteer/test_save_decisions.py]:
```suggestion
    for chip in artifact_chips:
        # clicking twice marks the artifact as absent
        await chip.click()
        await chip.click()
```

**Review by zachmullen — CHANGES_REQUESTED** [review]

**Review by zachmullen — APPROVED** [review]

**Pull request merged**